mlrun/utils/helpers.py

```diff
@@ -1007,6 +1007,10 @@ def get_workflow_url(project, id=None):
     return url
 
 
+def get_kfp_project_filter(project_name: str) -> str:
+    return f'{{"predicates": [{{"key": "name", "op": 9, "string_value": "{project_name}"}}]}}'
```

Comment:
Add explanation that we filter kfp runs using substring predicate as a heuristic to get a specific project runs.

Reply:
Also, it's better to use a variable than simply a number so we know what's going on here in the future.
https://github.com/kubeflow/pipelines/blob/master/backend/api/v1beta1/filter.proto#L43-L45

```suggestion
def get_kfp_project_filter(project_name: str) -> str:
    is_substring_op = 9
    return f'{{"predicates": [{{"key": "name", "op": {is_substring_op}, "string_value": "{project_name}"}}]}}'
```